src/cli/backup/backup.go

```diff
@@ -194,6 +194,8 @@ func runBackups(
 	r repository.Repository,
 	serviceName, resourceOwnerType string,
 	selectorSet []selectors.Selector,
+	resourceOwnersIDToName map[string]string,
+	resourceOwnersNameToID map[string]string,
```

Comment:
since this encapsulates a complete idea (the ability to go from ID->PN and vice-versa) and is passed through multiple layers of the stack at least make it a struct with a well defined interface. Ideally it would be an interface with a struct as the concrete implementation so that we could easily swap it out later without depending on the specific struct details

If you make it just a struct, please don't export the fields and use functions instead. Again, this is so we don't come to rely on specific struct internals and can swap it out more easily in the future if we need to